src/app/features/charts/components/median-barchart/median-barchart.component.ts

```diff
@@ -0,0 +1,260 @@
+    svg.style('display', 'block');
+  }
+
+  destroyChart() {
+    if (this.chartInitialized) this.chart.remove();
+    if (this.tooltipInitialized) this.tooltip.remove();
+  }
+
+  showTooltip(text: string, x: number, y: number): void {
+    this.tooltip = d3
+      .select(this.tooltipRef.nativeElement)
+      .style('left', `${x}px`)
+      .style('top', `${y}px`)
+      .style('display', 'block')
+      .html(text);
+    this.tooltipInitialized = true;
+  }
+
+  hideTooltip() {
+    if (this.tooltipInitialized) {
+      this.tooltip.style('display', 'none');
+    }
+  }
+
+  getBarCenterX(tissue: string, xScale: d3.ScaleBand<string>): number {
+    return (xScale(tissue) || 0) + xScale.bandwidth() / 2;
+  }
+
+  // get the current width allotted to this chart or default
+  getChartBoundingWidth(): number {
+    return (
+      d3.select(this.chartRef.nativeElement).node()?.getBoundingClientRect()
+        .width || 500
+    );
+  }
+
+  createChart() {
+    if (this._data.length > 0) {
+      const barColor = this.helperService.getColor('secondary');
+      const width = this.getChartBoundingWidth();
+      const height = 350;
+      const margin = { top: 20, right: 20, bottom: 65, left: 65 };
+      const innerWidth = width - margin.left - margin.right;
+      const innerHeight = height - margin.top - margin.bottom;
+
+      this.chart = d3
+        .select(this.chartRef.nativeElement)
+        .attr('width', width)
+        .attr('height', height)
+        .append('g')
+        .attr('transform', `translate(${margin.left}, ${margin.top})`);
+
+      // SCALES
+      const xScale = d3
+        .scaleBand()
+        .domain(this._data.map((d) => d.tissue))
+        .range([0, innerWidth])
+        .padding(0.2);
+
+      const yScale = d3
+        .scaleLinear()
+        .domain([0, this.maxValueY])
+        .nice()
+        .range([innerHeight, 0]);
+
+      // BARS
+      this.chart
+        .selectAll('.medianbars')
+        .data(this._data)
+        .enter()
+        .append('rect')
+        .attr('class', 'medianbars')
+        .attr('x', (d) => xScale(d.tissue) as number)
+        .attr('y', (d) => yScale(d.medianlogcpm || 0))
+        .attr('width', xScale.bandwidth())
+        .attr('height', (d) => innerHeight - yScale(d.medianlogcpm || 0))
+        .attr('fill', barColor);
+
+      // SCORE LABELS
+      this.chart
+        .selectAll('.bar-labels')
+        .data(this._data)
+        .enter()
+        .append('text')
+        .attr('class', 'bar-labels')
+        .attr('x', (d) => this.getBarCenterX(d.tissue, xScale))
+        .attr('y', (d) => yScale(d.medianlogcpm || 0) - 5)
+        .text((d) => this.helperService.roundNumber(d.medianlogcpm || 0, 2));
+
+      // X-AXIS
+      const xAxis = d3.axisBottom(xScale);
+      this.chart
+        .append('g')
+        .attr('class', 'x-axis')
+        .attr('transform', `translate(0, ${innerHeight})`)
+        .call(xAxis.tickSizeOuter(0))
```

Comment:
Remove outer x-axis ticks to more closely match the current implementation

Reply:
I love your screenshots, do you think you could add before / after for @JessterB?

Reply:
Sure! Here are some screenshots demonstrating the different x-axis tick options: 

without outer ticks* | with outer ticks
:---:|:---:
<img width="1377" alt="median-barchart-xAxisTicks-noOuterTick" src="https://github.com/Sage-Bionetworks/Agora/assets/26949006/27bc8f80-60c7-4c71-b9c1-c7aefa46231a"> | <img width="1392" alt="median-barchart-xAxisTicks-outerTicks" src="https://github.com/Sage-Bionetworks/Agora/assets/26949006/538c2f4a-a629-4f8b-9e9f-de5a7dc5413d"> 

*implementation currently in this PR